scripts/pre_push_hook.py

```diff
@@ -295,21 +295,20 @@ def _install_hook():
         print 'Copied file to .git/hooks directory'
 
 
-def does_diff_include_js_files(files_to_lint):
-    """Returns true if diff includes JS files.
+def does_diff_include_frontend_scripts(files_to_lint):
+    """Returns true if diff includes TypeScript or JavaScript files.
```

Comment:
May be alphabetize this :)
`TypeScript or JavaScript` --> `JavaScript or TypeScript`